app/assets/javascripts/components/util/history.js

```diff
@@ -1,26 +0,0 @@
-import { createBrowserHistory } from 'history';
-
-// Custom history component handles scroll position for back button, hashes,
```

Comment:
Did you find the situations where this was relevant, and test whether scroll position is still handled well during navigation without this scrollIntoView trigger?

Reply:
ah, I didn't actually see that. I'll check.